src/components/Search/style.scss

```diff
@@ -0,0 +1,4 @@
+.searchBox {
+    margin-left: 40%;
```

Comment:
are you sure that you need margin-left: 40%
you can set 'margin: 0 auto' if you want align you element to center

Reply:
component should not align itself. position of component should be aligned by parent one (App.tsx in your case).

Reply:
and read about `flexbox`